File: Mult_Digits/Mult.java
/*
    Auhtor : Pakshal Jain 
    Date : 20/10/2022
    Program : To Calculate Multiplication of Digits of Entered Number
*/

//Required Libraries
import java.lang.*;
import java.util.*;

//Class Declaration
class Calculate 
{
    //Characteristics of Class
    int No = 0;
    
    //Constructor of Class
    Calculate(int no)
    {
        this.No = no;
    }
    //Behaviour of Class
    int calcu()
    {
        int Digit = 0;
        int Mult = 1;

        //Logic
        while(No != 0)
        {
            Digit = this.No % 10;
            if(Digit == 0)
            {
                Digit = 1;
            }
            Mult = Mult * Digit;
            this.No = this.No / 10;
        }

        return Mult;
    }
}

//Main Class
class Mult
{
    public static void main(String args[])
    {
        int No = 0;
        int iret = 0;

        System.out.println("Jay Ganesh....");
        Scanner sobj = new Scanner(System.in);

        //Taking Input From User
        System.out.println("Enter One Number");
        No = sobj.nextInt();

        //Creating Object of Calculate Class
        Calculate cobj = new Calculate(No);
        iret = cobj.calcu();

        //Printing Multiplication of Digits
        System.out.println("Multiplication of Digits of Entered Number is : "+iret);
    }
}

File: ThreeNo_Multiplication/Mult.cpp
/*
    Author : Pakshal Jain
    Date : 12/10/2022
    Program : To Calulate Multiplication of Entered Three Number
*/

//Required Libraries
#include<iostream>
using namespace std;

//////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

//Class Decalaration
class Multiplication
{   
    //Chracteristics of Class
    private :
        int No = 0;
        int No2 = 0;
        int No3 = 0;
        int ans = 0;
    //Constructor of Class
    public :
        Multiplication(int no,int no2,int no3)
        {
            this->No = no;
            this->No2 = no2;
            this->No3 = no3;
        }
    //Behaviour of Class
    int Mult()
    {
        //Handeling Constions if any of the Number is Zero
        if((this->No == 0)&&(this->No2 == 0))
        {
            this->ans = this->No3;
        }
        else if((this->No2 == 0)&&(this->No3 == 0)) 
        {
            this->ans = this->No;
        }
        else if((this->No == 0)&&(this->No3 == 0))
        {
            this->ans = this->No2;
        }
        else if(this->No == 0)
        {
            this->ans = this->No2 * this->No3;
        }
        else if(this->No2 == 0)
        {
            this->ans = this->No * this->No3;
        }
        else if(this->No3 == 0)
        {
            this->ans = this->No * this->No2;
        }
        //Handeling Condition if any of the number is Not Zero
        else 
        {
            this->ans = this->No * this->No2 * this->No3;
        }

        return this->ans;
    }
};//End Of Class

/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

int main() 
{
    int No = 0;
    int No2 = 0;
    int No3 = 0;
    int iret = 0;

    cout<<"Jay Ganesh...\n";

    //Taking Input as Three Entered Numbers
    printf("Enter Numbers...\n");
    scanf("%d %d %d",&No,&No2,&No3);

    //Creating object of Class
    Multiplication mobj(No,No2,No3);

    //Handeling Condition if all The Entered Three Numbers are Zero
    if((No == 0)&&(No2 == 0)&&(No3 == 0))
    {
        iret = No * No2 * No3;
    }
    else 
    {
        //Calling Behviout of Class
        iret = mobj.Mult();
    }

    //Printting Final Output of Program
    printf("Multiplication of Numbers is : %d",iret);

    return 0;
}